src/csvcubed/utils/sparql_handler/sparqlmanager.py

```diff
@@ -367,6 +373,26 @@ def select_codelist_cols_by_dataset_url(
         )
     return map_codelist_cols_by_dataset_url_result(results)
 
+def select_primary_keys_by_dataset_url(
```

Comment:
Again, can you make it clear that you're selecting the primary key column names?